src/wm_info_provider/hyprland.rs

```diff
@@ -1,7 +1,6 @@
 use std::io::{self, Write};
 use std::os::fd::AsRawFd;
 use std::os::unix::net::UnixStream;
-
```

Comment:
Unrelated change. Also I like to have a blank line to separate std-imports from the rest.